covalent_braket_plugin/braket.py

```diff
@@ -81,33 +81,49 @@ class BraketExecutor(AWSExecutor):
 
     def __init__(
         self,
-        credentials: str,
-        profile: str,
         s3_bucket_name: str,
```

Comment:
Hi @AlejandroEsquivel, your change is a great idea and will help further decouple the client-side and server-side config. Can we also assign default values to the remaining parameters `s3_bucket_name`, `ecr_repo_name`, `braket_job_execution_role_name`?